src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -112,5 +112,26 @@
     </table>
   </div>
 </div>
+<div class="section">
+  <div class="section-heading">
+    <strong>Declaration</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr>
+        <td></td>
+        <td>{{ ccj.claimantName}}</td>
```

Comment:
do we not need the company name and position here?

Reply:
not as per https://tools.hmcts.net/jira/browse/ROC-2468

((Claimant name)) [in future the person signing the declaration could be someone other than the claimant, but for now it's only this option]